flow/project.py

```diff
@@ -3105,10 +3120,27 @@ def _run_operations_in_parallel(self, pool, operations, progress, timeout):
             pool.apply_async(_deserialize_and_run_operation, task)
             for task in serialized_tasks
         ]
-        for result in tqdm(results) if progress else results:
+        if progress:
+            results = tqdm(results)
+        for result in results:
```

Comment:
This rewrite is equivalent. I chose this pattern instead because it matches other parts of the code and slightly improves readability.